tests/src/Test.idr

```diff
@@ -30,10 +30,11 @@ main = run
   , run_test "decompress jabberwock.txt.gz" test_gzip_text
   , run_test "decompress jabberwocky.jpg.gz" test_gzip_jpg
   , run_test "decompress concatenated.gz" test_gzip_concated
-  , run_test "http close w/out read" test_close_without_read
+  --, run_test "http close w/out read" test_close_without_read
   , run_test "http cookie jar" test_cookie
   , run_test "http httpbin deflate" test_json_deflate
   , run_test "http httpbin gzip" test_json_gzip
   , run_test "http httpbin post" test_post
   , run_test "http openbsd redirect" test_redirect
+  --, run_test "chunked transfer encoding" test_chunked_transfer_encoding
```

Comment:
bugger chunked response encoding doesnt work / is not supported

Reply:
which perfectly explains why i couldnt make calls to discord api from by idris2 bot (small pet project)